server/routes/members-routes.js

```diff
@@ -10,8 +10,12 @@ router.use((req, res, next) => {
 
 router.get('/all', all);
 
-router.post('/', create);
+router.post('/create', create);
```

Comment:
This is not clear. By convention POST is used to create new things. What new thing is being created here? Also the other routes are similarly not well-designed. Please think about `collections` of things. For example, you might have users, hence a router.post() to create a new user and a `users.put` for updating user info.

Reply:
This is a new member bring created because we are in the member routes. We have similar set up for squads and ranks.